docs/HOWTO_HEROKU_DEPLOY.md

```diff
@@ -16,23 +18,34 @@
   - Domain name = AUTH0_DOMAIN
   - Client ID = AUTH0_CLIENT_ID
   - Client Secret = AUTH0_CLIENT_SECRET
-- These variables should be placed in your heroku environment variables form
-- Scroll to `Allowed Callback URLs` section and update it with (your heroku_app_url):
+- These variables should be placed in your Heroku configuration variables form
+- Scroll to `Allowed Callback URLs` section and update it with (your HEROKU_APP_URL):
   - `https://<YOUR_HEROKU_APP_URL>/login-callback, http://<YOUR_HEROKU_APP_URL>/login-callback`
 
-- Scroll to `Allowed Logout URLs` section and update it with (your heroku_app_url):
-  - `https://<YOUR_HEROKU_APP_URL>/login-callback, http://<YOUR_HEROKU_APP_URL>/login-callback`
-
-  - `https://<YOUR_HEROKU_APP_URL>/login-callback` = AUTH0_LOGIN_CALLBACK
-  - `https://<YOUR_HEROKU_APP_URL>/logout-callback` = AUTH0_LOGOUT_CALLBACK
+- Scroll to `Allowed Logout URLs` section and update it with (your HEROKU_APP_URL):
+  - `https://<YOUR_HEROKU_APP_URL>/logout-callback, http://<YOUR_HEROKU_APP_URL>/logout-callback`
+- As a note:
+  - AUTH0_LOGIN_CALLBACK in your config variables is the same as `https://<YOUR_HEROKU_APP_URL>/login-callback`
+  - AUTH0_LOGOUT_CALLBACK in your config variables is the same as `https://<YOUR_HEROKU_APP_URL>/logout-callback`
 - Scroll to `Allowed Origin (CORS)` add:
-  - ` http://*.herokuapp.com`, ` https://*.herokuapp.com`
+  - ` http://*.<YOUR_HEROKU_APP_URL>.com`, ` https://*.<YOUR_HEROKU_APP_URL>.com`
 - Scroll to bottom and click on `Advanced Settings`
 - Click on `OAuth` - make sure `OIDC Conformant` is turned off.
-
-
-## Notes about Twilio environment variable setup
+- Then create a rule in Auth0:
```

Comment:
I guess just duping the instructions from README is good for now, but it would be good to figure out how to have these in one place.  Should both say "goto docs/HOWTO_setup_auth0.md and when you finish come back here"?